openapi/SwarmDebug.yaml

```diff
@@ -1150,3 +1150,20 @@ paths:
           $ref: "SwarmCommon.yaml#/components/responses/400"
         default:
           description: Default response
+
+  "/status":
+    get:
+      summary: Get all available loggers that match the specified expression.
```

Comment:
It looks to me that this description is not correct.

Reply:
Nice catch. Fixed!